examples/namespace_declaration.cf

```diff
@@ -0,0 +1,64 @@
+# By default we are in the default namespace
+
+bundle agent __main__
+{
+  methods:
+      "Main in My_Namespace namespace"
+        usebundle => My_Namespace:main;
```

Comment:
I think using capitalized `My_Namespace` can be a distraction / confusing. Should be `my_namespace`:

```suggestion
      "Main in my_namespace namespace"
        usebundle => my_namespace:main;
```

Reply:
See what anyone else has to say, if someone else likes the variety ill keep it, if no one like the variety then i wont.

Reply:
@craigcomstock @vpodzime what say you about this?

Reply:
I actually wish we had Camel_Case namespaces everywhere and it was recommended.

Reply:
> I actually wish we had Camel_Case namespaces everywhere and it was recommended.

But you didn't :+1: this PR so i still have to make changes to get it merged.